ui/src/main/webapp/adminTools/webContextPolicyManager/actions.js

```diff
@@ -42,7 +42,7 @@ export const addContextPath = (attribute, binNumber) => (dispatch, getState) =>
   if (!newPath || newPath.trim() === '') { return }
 
   // simple test for invalid path - backend also validates paths as an additional precaution
-  if (!(/^(\/[!#$&-;=?-[\]_a-z~]+)+/.test(newPath))) {
+  if (!(/^(\/[!#$&-;=?-[\]_a-z~]*)+/.test(newPath))) {
```

Comment:
1. I see this is currently in only one Javascript file but imagine this regex _might_ be more generally applicable. If so, it would be good to extract it as a function.
2. It is an especially ugly regex and would benefit from a comment explaining it, possibly with examples that would pass and examples that would fail.
3. That comment would be less intrusive if it were on an extracted function. 😄